lms/templates/courseware/progress.html

```diff
@@ -18,6 +18,7 @@
 <%namespace name="progress_graph" file="/courseware/progress_graph.js"/>
 
 <%block name="pagetitle">${_("{course_number} Progress").format(course_number=course.display_number_with_default) | h}</%block>
+<%block name="nav_skip">#content</%block>
```

Comment:
I intentionally removed these blocks to rely on hard-coded and same skip links. They all point to `id="main"`, which should be the `main` element. Adding this in will cause text to appear at the top of the page.